php/classes/controllers/class-feed-controller.php

```diff
@@ -316,10 +316,10 @@ public function load_feed_template() {
 		}
 		$explicit_option = apply_filters( 'ssp_feed_explicit', $explicit_option, $series_id );
 		if ( $explicit_option && 'on' === $explicit_option ) {
-			$itunes_explicit     = 'yes';
+			$itunes_explicit     = 'true';
```

Comment:
This is wtat's wanted by W3 Validator. Never ever "yes" and "clean" but "true" and "false"